www/privacy/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page 
+title: Privacy Policy 
+---
+
+# Privacy Policy
+
+When you opt into telemetry, information about your usage of Apache Cordova CLI is collected. The collected information consists of the following:
+
+* IP address
+* OS type and version
+* Node version
+* Cordova version
+* Commands executed, time of execution and status (success/failure)
+
+This information is saved in Google Analytics and handled by Google as described in their [privacy policy](http://www.google.com/policies/privacy/). See the documentation for instructions on how to disable telemetry if you prefer not to share your data.
```

Comment:
We should probably link to http://cordova.apache.org/docs/en/latest/reference/cordova-cli/index.html and add the docs for it.